client/src/components/TestSuite/TestSession.tsx

```diff
@@ -220,8 +220,8 @@ const TestSessionComponent: FC<TestSessionComponentProps> = ({
       });
   }
 
-  function testRunNeedsProgressBar(testRun: TestRun | null) {
-    return testRun?.status && ['running', 'queued', 'waiting'].includes(testRun.status);
+  function testRunNeedsProgressBar(testRun: TestRun | null): boolean {
+    return testRun?.status ? ['running', 'queued', 'waiting'].includes(testRun?.status) : false;
```

Comment:
Minor point, but I think the original formulation of `A && B` is more conventional and easier to follow than `A ? B : false`.  Unless I'm getting my logic wrong, and they aren't the same.  Or maybe a linter encourages the second form?

Reply:
I agree with you that the first is easier to understand - I ended up making this change because the return type of testRunNeedsProgressBar in the first form was {boolean | undefined}, and I think it makes sense to check things as tightly as possible, especially because the places where we call testRunNeedsProgressBar would end up interpreting an undefined return as a boolean anyways. So, not a super important change and doesn't make any actual difference to me - I'm good with either.

Reply:
Ahhh, ok, this is so that Typescript can trust that the return is truly a bool.  That's a great reason for using this form over `A&&B`.  I'm good with this then.